Packs/AzureSecurityCenter/Integrations/MicrosoftDefenderForCloudEventCollector/test_data/AlertsToSort.json

```diff
@@ -0,0 +1,175 @@
+[
+  {
+    "id": "/subscriptions/0f907ea4-bc8b-4c11-9d7e-805c2fd144fb/resourceGroups/Sample-RG/providers/Microsoft.Security/locations/centralus/alerts/2517396773228188817_67a23793-595b-4efb-ae23-679568966575",
```

Comment:
Is it a real ID ? Wit hthe subscription ?

Reply:
No its a Microsoft fake generated data